docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
-
-
-@sphinxdirective
-   
-.. tab:: IR
-
-   .. code-block:: c
-
-      executable_network = core.LoadNetwork("model.xml", "CPU");
-
-.. tab:: ONNX
-
-   .. code-block:: c
-
-      executable_network = core.LoadNetwork("model.onnx", "CPU");
-
-.. tab:: nGraph
+#### Step 3. Compile the Model
 
-   .. code-block:: c
+Compile the model to the device using `ov::Core::compile_model()`:
 
-      std::shared_ptr<Function> createNetwork() {
-         // To construct a network, please follow 
-         // https://docs.openvino.ai/latest/openvino_docs_nGraph_DG_build_function.html
-      }
-      auto network = CNNNetwork(createNetwork());
-      executable_network = core.LoadNetwork(network, "CPU");
-
-.. tab:: Model From Step 2
-   
-   Follow this step only if you went through optional "Step 2 (Optional). Configure Input and Output of the Model", otherwise use another tab for your model type: IR (OpenVINO Intermediate Representation), ONNX or nGraph.
-   
-   .. code-block:: c
+   - IR:
+        @snippet snippets/src/main.cpp part4_1
+   - ONNX:
+        @snippet snippets/src/main.cpp part4_2
+   - Paddle:
+        @snippet snippets/src/main.cpp part4_3
+   - OpenVINO Model:
+        @snippet snippets/src/main.cpp part4_4
 
-      executable_network = core.LoadNetwork(network, "CPU");
 
-@endsphinxdirective
-
-
-It creates an executable network from a network object. The executable network is associated with single hardware device.
+It creates a compiled model from a model object. The compiled model is associated with single hardware device.
 It is possible to create as many networks as needed and to use them simultaneously (up to the limitation of the hardware resources).
 
 Third parameter is a configuration for plugin. It is map of pairs: (parameter name, parameter value). Choose device from
 [Supported devices](supported_plugins/Supported_Devices.md) page for more details about supported configuration parameters.
 
-@snippet snippets/Integrate_with_customer_application_new_API.cpp part6
+@snippet snippets/src/main.cpp part5
 
 #### Step 4. Create an Inference Request
 
 Create an infer request using the following code:
 
-@snippet snippets/Integrate_with_customer_application_new_API.cpp part7
+@snippet snippets/src/main.cpp part6
 
 #### Step 5. Prepare Input 
 
 You can use one of the following options to prepare input:
 
-* **Optimal way for a single network.** Get blobs allocated by an infer request using `InferenceEngine::InferRequest::GetBlob()` and feed an image and the input data to the blobs. In this case, input data must be aligned (resized manually) with a given blob size and have a correct color format.
+* **Optimal way for a single network.** Get tensor allocated by an infer request using `ov::InferRequest::get_tensor()` and feed an image and the input data to the tensors. In this case, input data must be aligned (resized manually) with a given blob size and have a correct color format.
```

Comment:
- single network => single model
- feed an image => feed input tensor
- `In this case, input data must be aligned (resized manually) with a given blob size and have a correct color format` => `Input tensor's shape, element type must match specific input of the model. For cases of dynamic input shapes, read [Working with dynamic shapes]`